### PR TITLE
update documentation 

### LINKED ISSUE #7: metaT assembly - Kaitlyn
Follow the template below for the index.rst file that is in the docs folder of the repo for your workflow:
Title of Workflow
(Image)
Workflow overview
Workflow availability
Requirements for Execution
Hardware requirements
Workflow dependencies
Software
Database(s)
Sample dataset(s)
Input
Output
Version history
Point of contact

If you have updated figures please put the editable components of the figures [here](https://drive.google.com/drive/folders/1MEmTMLMPeDMQaF-LYnpJswjVmn9MUFTX) as well as in the documentation 
Please follow the following format for updating your documentation:

### PR DESCRIPTION
**Pull request by kaijli**
added image and updated index.rst

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # The Metatranscriptome Assembly Pipeline
 
 ## Summary
-This workflow is developed by Brian Foster at JGI. Original repo can be found [here](https://code.jgi.doe.gov/BFoster/jgi_meta_wdl/-/tree/master/metatranscriptome). This workflow uses BBTools and SPAdes to assemble QC'ed transcriptomic reads. 
+This workflow was developed by Brian Foster at JGI. Original repo can be found (here)[https://code.jgi.doe.gov/BFoster/jgi_meta_wdl/-/tree/master/metatranscriptome]. This workflow uses BBTools and SPAdes to assemble QC'ed transcriptomic reads.
```
Comment (aclum) [inline README.md]: check with Brian F., this is out of date since isn't on spades version 4.

Comment (aclum) [inline README.md]: here hyperlink format is not correct.

File: docs/mt_assy_workflow2024.png
Comment (aclum) [inline docs/mt_assy_workflow2024.png:1]: can this look more like what we have for metagenome assembly https://nmdc-workflow-documentation.readthedocs.io/en/latest/chapters/3_MetaGAssemly_index.html

File: docs/index.rst
Comment (aclum) [inline docs/index.rst:1]: formatting issues on this file.

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Third party software (This is included in the Docker images.)  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `BBTools v38.96 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `SPAdes v3.15.4 <https://github.com/ablab/spades>`_ (License: `SPAdes team <https://github.com/ablab/spades?tab=License-1-ov-file#License-1-ov-file>`_)
+
+
+Sample datasets
+---------------
+- Processed Metatranscriptome of soil microbial communities from the East River watershed near Crested Butte, Colorado, United States - ER_RNA_119 (`SRR11678315 <https://www.ncbi.nlm.nih.gov/sra/SRX8239222>`_) with `metadata available in the NMDC Data Portal <https://data.microbiomedata.org/details/study/nmdc:sty-11-dcqce727>`_.
```
Comment (aclum) [inline docs/index.rst:52]: subbullet formatting issues.

File: docs/mt_assy_workflow2024.png
Comment (kaijli) [reply docs/mt_assy_workflow2024.png:1]: mt_assy
how about this?

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+
+Workflow Dependencies
+---------------------
+
+Third party software (This is included in the Docker images.)  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `BBTools v38.96 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `SPAdes v3.15.4 <https://github.com/ablab/spades>`_ (License: `SPAdes team <https://github.com/ablab/spades?tab=License-1-ov-file#License-1-ov-file>`_)
```
Comment (aclum) [inline docs/index.rst]: update text from SPAdes team to GNU General Public License, Version 2, hyperlink is good.

File: docs/mt_assy_workflow2024.png
Comment (aclum) [reply docs/mt_assy_workflow2024.png:1]: yes, please update to this.

**Comment by aclum** [comment]
I was suprised that the end to end workflow doesn't include read qc?

**Comment by kaijli** [comment]
the scripts that are run are `readlength.sh`, `rnaspades.py`, `fungalrelease.sh`, `bbstats.sh`, `bbmap.sh`, `samtools index`, `reformat.sh`, `pileup.sh`, and `samtools flagstat`. I don't think any of those are read qc tools?

**Comment by aclum** [comment]
They aren't but this is an oversight IMO.  The end to end metagenome workflow contains a filtering step https://github.com/microbiomedata/metaG/blob/main/nmdc-metag_full.wdl  (call rqc.jgi_rqcfilter on line 26). The metaT end to end workflow should mirror this and have a filtering (read qc) step.

**Comment by kaijli** [comment]
is readsqc not enough to count towards that? this is the assembly ticket?

File: README.md [hunk 4]
```diff
@@ -29,8 +30,8 @@ The inputs for this workflow are as follows:
 
 ```
 {
-  "metatranscriptome_assy.input_files":["./test_data/nmdc_xxxxxxx_filtered.fastq.gz"],
-  "metatranscriptome_assy.proj_id":"nmdc:xxxxxxx"
+  "metatranscriptome_assy.input_files":["https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/readsqc_output/SRR11678315-int-0.1_filtered.fastq.gz"],
+  "metatranscriptome_assy.proj_id":"nmdc_xxxxxxx"
```
Comment (aclum) [inline README.md]: okay to leave like this but for production it the proj_id should be "nmd:xxx"

**Review by aclum — APPROVED** [review]

**Pull request merged**